src/main/java/Backend/Expressions/ArithmeticOperatorExpression.java

```diff
@@ -16,6 +18,24 @@ public ArithmeticOperatorExpression(String operation, RealValuedExpression lExpr
         this.lExpression = lExpression;
         this.rExpression = rExpression;
     }
+    @Override
+    public String toString() {
+        StringBuilder str = new StringBuilder();
+        if (getItem().equals("^")) {
+            str.append("pow(");
+            str.append(lExpression);
+            str.append(",");
+            str.append(rExpression);
```

Comment:
Could you not use String concatenation here as well rather than have a StringBuilder object?

Reply:
Actually, yea that is possible. Just have a return statement inside the if block. Dont know why I didn't do that